src/util/Injected.js

```diff
@@ -176,21 +176,27 @@ exports.LoadCustomSerializers = () => {
 
     window.WWebJS.getChatModel = chat => {
         let res = chat.serialize();
+
+        res.kind = chat.kind;
         res.isGroup = chat.isGroup;
         res.formattedTitle = chat.formattedTitle;
 
         if(chat.groupMetadata) {
             res.groupMetadata = chat.groupMetadata.serialize();
         }
 
+        if (!res.isGroup) {
```

Comment:
This `if` here isn't really needed since all chat objects have a presence, even though it will always be `false` for groups.